InsightsFrontend/src/app/components/tab-header/tab-header.component.html

```diff
@@ -0,0 +1,17 @@
+<nav id="tab-header">
+  <img ngSrc="./assets/logo.png" alt="FF! insights Logo" width="48" height="48" priority />
```

Comment:
The navigation style should follow are design language. That is: Use the Frank!Framework logo, and add your application name next to it. Some examples:

![image](https://github.com/user-attachments/assets/68cd799b-c716-4674-9ee9-eb755e376cf9)
![image](https://github.com/user-attachments/assets/3d80e647-c326-4be9-a482-4d425653ac8a)
![image](https://github.com/user-attachments/assets/d8c28ca0-cb31-4153-9237-bf1a48b6f191)